ui/hooks/useName.ts

```diff
@@ -23,6 +23,8 @@ function getVariationKey(type: string, chainId: string): string {
   }
 }
 
+export const FALLBACK_VARIATION = '*';
```

Comment:
We can't reference this in our background classes as we want the UI and background to be decoupled.

We could define it in the `shared` directory but since we know this will ultimately be defined in the `NameController`, we could also just define it here and in the bridge.

Reply:
Done.